docs/linux_dependencies.md

```diff
@@ -45,48 +45,44 @@ sudo xbps-install -S pkgconf alsa-lib-devel libX11-devel eudev-libudev-devel
 
 ## NixOS
 
-Add a `build.rs` file to your project with the following:
+Add a `build.rs` file to your project:
 
 ```rust
+$ cat > build.rs <<EOF
 fn main() {
     if cfg!(target_os = "linux") {
         println!("cargo:rustc-link-lib=vulkan");
     }
 }
+EOF
```

Comment:
I switched to using heredocs so all instructions can be simply copied into terminal without paying too much attention to what needs to be put into some file and what needs to be executed in shell.

Reply:
I'd rather just keep it as file contents myself.

Reply:
I'll change that